src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,193 @@
+package duke;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Scanner;
+import java.util.ArrayList;
```

Comment:
Might want to consider breaking up the imports into groups (ie. leave a space after line 3) to follow the coding standards?